docs/events/indicators-of-compromise.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[indicators-of-compromise]]
+= Indicators of compromise
+
+The Intelligence page collects data from enabled threat intelligence feeds and provides you with a centralized view of indicators. This topic offers definitions of threat intelligence and indicators, helps you get started using the Intelligence page, and explains how to work with indicators.
+
+[role="screenshot"]
+image::images/indicators-table.png[Shows the Indicators table on the Intelligence page]
+
+[discrete]
+[[ti-indicators]]
+== Threat intelligence and indicators
+If you are unfamiliar with threat intelligence, think of it as a research function that helps an organization make informed decisions when taking actions to protect itself. An organization's security operation center (SOC) team is usually responsible for managing threat intelligence. A SOC team's goals are to understand current and emerging threats so they can recommend actions that will prevent, stop, and mitigate threats to an organization.
+
+An indicator, also referred to as an Indicator of Compromise (IoC), is a piece of information that represents a known malicious threat or reported vulnerability. There are many types of indicators, including URLs, files, domains, email addresses and more. Within SOC teams, threat intelligence analysts use indicators to detect, assess, and respond to threats.
+
+[discrete]
+[[setup-intelligence-page]]
+== Set up the Intelligence page
+
+Install threat intelligence integrations to add indicators to the Intelligence page.
+
+IMPORTANT: Before enabling a threat intelligence feed, ensure you have installed a {fleet-guide}/install-fleet-managed-elastic-agent.html[{fleet}-managed {agent}] on the hosts you want to monitor.
+
+. Choose one of the following:
+.. From the {security-app} main menu, click the Intelligence page, then click *Add Integrations*.
+.. From the {kib} main menu, click *Add integrations*. Scroll down the list of integration categories and select *Threat Intelligence* to filter for threat intelligence integrations.
```

Comment:
Change to bullets, since these are separate options, not sequential steps. I _think_ this should work (per [Asciidoc docs](https://docs.asciidoctor.org/asciidoc/latest/lists/ordered/#nested-ordered-list)) -- also tightening up menu options in the first bullet:
```suggestion
* From the {security-app} main menu, click *Intelligence* -> *Add Integrations*.
* From the {kib} main menu, click *Add integrations*. Scroll down the list of integration categories and select *Threat Intelligence* to filter for threat intelligence integrations.
```

Reply:
Oh yeah, good catch - thanks!